CoreDependencies/src/ivy.xml

```diff
@@ -2,34 +2,9 @@
 <ivy-module version="2.0" xmlns:e="http://ant.apache.org/ivy/extra">
     <info organisation="au.gov.asd.tac" module="tac-utilities" revision="working"/>
     <dependencies>
-        <!-- Constellation Dependencies -->
```

Comment:
@cygnus-x-1 nice work updating the version numbers. Would you be able to mention what libraries and version numbers you've upgraded in `ivy.xml` by documenting this in `CHANGELOG.md`? It is difficult to understand which version numbers have changed and knowing this is going to be important for anyone that depends on these Core modules.

Reply:
Done.